cmd/manager/main.go

```diff
@@ -142,7 +144,6 @@ func main() {
 	if err = serveCRMetrics(cfg); err != nil {
 		log.Info("Could not generate and serve custom resource metrics", "error", err.Error())
 	}
-
```

Comment:
could you undo these while you're at it?

Reply:
I don't see any change here

Reply:
dene

Reply:
I still see this one:
![image](https://user-images.githubusercontent.com/57443811/119849711-8619fa80-bf15-11eb-929d-993f68585d9e.png)